yaksh/test_models.py

```diff
@@ -143,24 +149,33 @@ def test_dump_questions_into_json(self):
         """ Test dump questions into json """
         question = Question()
         question_id = [self.question2.id]
-        questions = json.loads(question.dump_into_json(question_id, self.user2))
+        questions_zip = question.dump_questions(question_id, self.user2)
+        zip_file = zipfile.ZipFile(questions_zip, "r")
+        zip_file.extractall("/tmp/")
```

Comment:
Never do this, instead use `tempfile` to create a temporary directory and then clear out that directory in `tearDown`.